docs/learn-accounts.md

```diff
@@ -531,6 +531,45 @@ networks.
 Alternatively, use [this handy subscan tool](https://polkadot.subscan.io/tools/ss58_transform) or
 [this simple address address convertor](https://polkadot-address-convertor.netlify.app/).
 
+### How to Verify a Public Key's Associated Address
+
+You can verify your public key's associated address through a series of conversion steps, where 
+the key is a base-16 (hexadecimal) address. Start by prefixing your public key with "**0x**" 
+so it can be interpreted as a hexadecimal string.
+
+#### Consider the following example:
+
+![17](assets/accounts/pubkey-1.png)
+
+From: `Pay DOTs to the Polkadot account:192c3c7e5789b461fbf1c7f614ba5eed0b22efc507cda60a5e7fda8e046bcdce`, 
+we prefix the address by "0x" -> `0x192c3c7e5789b461fbf1c7f614ba5eed0b22efc507cda60a5e7fda8e046bcdce`.
+
+Now, your prefixed string can use [Substrate Utilities](https://www.shawntabrizi.com/substrate-js-utilities/) 
```

Comment:
Do we want to link to something as unofficial as Shawn's playground? It's useful but not sure how permanent, long term. This isn't super simple, so I think maybe the same learning curve could be applied to teaching people who to verify this with Polkadot's CLI tools.

Reply:
Right, I see what you mean. @laboon what do you think?

Reply:
We could replace this with subkey, or even better yet, add an equivalent JavaScript widget to the Wiki. That would be ideal, I think.

```
subkey inspect 0x192c3c7e5789b461fbf1c7f614ba5eed0b22efc507cda60a5e7fda8e046bcdce
```

should work, although I'm in the process of rebuilding subkey on my machine right now so can't double-check...

Reply:
> an equivalent JavaScript widget to the Wiki

That's what I'd like to make as part of #1919

Reply:
Sorry if I misunderstood, did we want to use subkey or leave Shawn's playground short-term until the proposed UX improvements (which would include a widget I assume).

Reply:
I ended up using the conversion tools already listed on the page for "conversion purposes", since I don't think this can be done using subkey and I was trying to figure out the best approach for "verification" while still using the inspection command subkey offers.

Reply:
Subkey does it pretty easily. Here I do it first with a Substrate (default) address and then with Polkadot.

```
laboon@billcomp Downloads % subkey inspect 0x192c3c7e5789b461fbf1c7f614ba5eed0b22efc507cda60a5e7fda8e046bcdce

Secret Key URI `0x192c3c7e5789b461fbf1c7f614ba5eed0b22efc507cda60a5e7fda8e046bcdce` is account:
  Secret seed:      0x192c3c7e5789b461fbf1c7f614ba5eed0b22efc507cda60a5e7fda8e046bcdce
  Public key (hex): 0x5a095388156e3a68d4fb7cbaef981478e1a0be6d4998f00dffc3e4e9c60c104c
  Account ID:       0x5a095388156e3a68d4fb7cbaef981478e1a0be6d4998f00dffc3e4e9c60c104c
  SS58 Address:     5E6kwKEhrpVMnZvkBRFCzCcRnMXcft4HSaogYQtgtaw6QJ5s
laboon@billcomp Downloads % subkey inspect 0x192c3c7e5789b461fbf1c7f614ba5eed0b22efc507cda60a5e7fda8e046bcdce --network polkadot

Secret Key URI `0x192c3c7e5789b461fbf1c7f614ba5eed0b22efc507cda60a5e7fda8e046bcdce` is account:
  Secret seed:      0x192c3c7e5789b461fbf1c7f614ba5eed0b22efc507cda60a5e7fda8e046bcdce
  Public key (hex): 0x5a095388156e3a68d4fb7cbaef981478e1a0be6d4998f00dffc3e4e9c60c104c
  Account ID:       0x5a095388156e3a68d4fb7cbaef981478e1a0be6d4998f00dffc3e4e9c60c104c
  SS58 Address:     13345eVmibkqE6wG94JD8MSadyXGNBcRX5YAhht3Sfxcaw9U

```